orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -27,6 +27,11 @@
 #include "orttraining/core/framework/torch/custom_function_register.h"
 #endif
 
+#if defined(ENABLE_TRAINING) && defined(ENABLE_TRAINING_ON_DEVICE)
```

Comment:
just curious: given this file is specifically for training do we need defined(ENABLE_TRAINING) ?

Reply:
No, we only need `defined(ENABLE_TRAINING_ON_DEVICE)`. I'll make the change.

Reply:
Please address other occurrences in this file too. Thanks!